pkg/models/temp_user.go

```diff
@@ -82,24 +82,44 @@ type GetTempUsersQuery struct {
 }
 
 type GetTempUserByCodeQuery struct {
-	Code string
-
+	Code   string
 	Result *TempUserDTO
 }
 
+type Time struct{ time.Time }
 type TempUserDTO struct {
-	Id             int64          `json:"id"`
-	OrgId          int64          `json:"orgId"`
-	Name           string         `json:"name"`
-	Email          string         `json:"email"`
-	Role           org.RoleType   `json:"role"`
-	InvitedByLogin string         `json:"invitedByLogin"`
-	InvitedByEmail string         `json:"invitedByEmail"`
-	InvitedByName  string         `json:"invitedByName"`
-	Code           string         `json:"code"`
-	Status         TempUserStatus `json:"status"`
-	Url            string         `json:"url"`
-	EmailSent      bool           `json:"emailSent"`
-	EmailSentOn    time.Time      `json:"emailSentOn"`
-	Created        time.Time      `json:"createdOn"`
+	Id             int64          `json:"id" db:"id"`
+	OrgId          int64          `json:"orgId" db:"org_id"`
+	Name           string         `json:"name" db:"name"`
+	Email          string         `json:"email" db:"email"`
+	Role           org.RoleType   `json:"role" db:"role"`
+	InvitedByLogin *string        `json:"invitedByLogin" db:"invited_by_login"`
+	InvitedByEmail *string        `json:"invitedByEmail" db:"invited_by_email"`
+	InvitedByName  *string        `json:"invitedByName" db:"invited_by_name"`
+	Code           string         `json:"code" db:"code"`
+	Status         TempUserStatus `json:"status" db:"status"`
+	Url            string         `json:"url" db:"url"`
+	EmailSent      bool           `json:"emailSent" db:"email_sent"`
+	EmailSentOn    *time.Time     `json:"emailSentOn" db:"email_sent_on"`
```

Comment:
I have some concerns about going from time.Time to *time.Time as well. I expect that fine, I haven't found any callers that use that information, but we may want to at least keep an eye out for issues.